content/blog/2016/2016-09-27-jenkins-world-2016-wrap-up-scaling.adoc

```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+title: "Jenkins World 2016 Wrap-up - Scaling"
+tags:
+- event
+- jenkinsworld
+author: lnewman
+---
+
+NOTE: This is a guest post by link:https://github.com/bitwiseman[Liam Newman],
+Technical Evangelist at CloudBees.
+
+image:/images/conferences/Jenkins-World_125x125.png[Jenkins World 2016, role=right]
+
+I've used Jenkins for several years and been a contributor for the last two, but
+this was my first time at a Jenkins user conference.
```

Comment:
If this is to be part of the lead-in, it seems a bit awkward. This being your first time at a Jenkins conference (IMHO we should avoid "Jenkins user conference" since "JUC" doesn't exist anymore) doesn't relate to Scaling Jenkins.

As long-time Jenkins users, we've certainly grown Jenkins clusters (IIRC), but the cool thing (IMO) about these presentations at Jenkins World is seeing how far these organizations are taking it.